Tasks/DockerComposeV0/dockercomposebuild.ts

```diff
@@ -5,30 +5,32 @@ import DockerComposeConnection from "./dockercomposeconnection";
 import * as sourceUtils from "docker-common-v2/sourceutils";
 import * as imageUtils from "docker-common-v2/containerimageutils";
 import * as dockerCommandUtils from "docker-common-v2/dockercommandutils";
+import * as utils from "./utils";
 
-function dockerTag(connection: DockerComposeConnection, source: string, target: string) {
+function dockerTag(connection: DockerComposeConnection, source: string, target: string, outputUpdate: (data: string) => any) {
     var command = connection.createCommand();
     command.arg("tag");
     command.arg(source);
     command.arg(target);
-    return connection.execCommand(command);
+    return connection.execCommandWithLogging(command)
+    .then((output) => outputUpdate(utils.writeTaskOutput("tag", output)));
 }
 
-function addTag(promise: any, connection: DockerComposeConnection, source: string, target: string) {
+function addTag(promise: any, connection: DockerComposeConnection, source: string, target: string, outputUpdate: (data: string) => any) {
```

Comment:
I'm not in favour of having this callback as an input, cause this can be set here itself. 
What are your thoughts?

Reply:
I've added this as an input so that we can add the name of the image to the tag. For example if I'm tagging multiple images the output would look like this:

```
/home/vsts/work/_temp/task_outputs/tag_image1_1586797766277.log
/home/vsts/work/_temp/task_outputs/tag_image2_1586797766278.log
```

I can also return logging from the addTag function, but I don't think it makes a lot of sense to have an `addTag` method return the plain logging of the docker-compose command. I would rather expect it to return some kind of new image name. So to avoid confusion here I'd rather implement it in a way that we can be sure this contains the output of the build.

Reply:
I rewrote this, could you look at my changes?